docker-compose-yml-gen.py

```diff
@@ -0,0 +1,89 @@
+import secrets
+import string
+from typing import Optional
+
+import yaml
+
+HOST_PORT_DEFAULT = 80
+
+
+def generate_secret_key() -> str:
+    """Generate securely random 50 ASCII letters and digits."""
+    return "".join(
+        secrets.choice(string.ascii_letters + string.digits) for _ in range(50)
+    )  # random 50 ASCII letters and digits
+
+
+def save_yml(*, host_port: int, secret_key: str):
+    with open("docker-compose.yml", "w") as file:
+        yaml.dump(
```

Comment:
This could just overwrite the prebaked `docker-compose.yml` that ships with the repo right?

Reply:
It'd be a bit more brittle, but it could